packages/dx-react-demos/src/demo-sources/grid-data-types/material-ui/editors.jsx

```diff
@@ -31,7 +31,7 @@ const BooleanEditor = ({ value, onValueChange }) => (
     input={<Input />}
     value={value ? 'Yes' : 'No'}
     onChange={event => onValueChange(event.target.value === 'Yes')}
-    style={{ width: '100%' }}
+    style={{ width: '100%', marginTop: '4px' }}
```

Comment:
Is not it possible to calculate this magic '4px' value somehow?

Reply:
It's mui bug, and we add this style because don't have `const theme` here

Reply:
Yes, it is. It's kind of 'fix after update MUI':) So, we decided to keep code simple and not to use `withStyles` etc.